### PR TITLE
[POST] Abigail Mesrenyame Dogbe Honored with Outstanding PyLady Award…

### LINKED ISSUE #330: Announcement Post - Abigail Mesrenyame Dogbe Wins Inaugural Outstanding PyLady Award
Create Announcement Post with the following Format

- the announcement
- what is Pyladies
- what is the outstanding pylady award
- when was the award given (On stage at PyCon 2024 in Pittsburgh,PA. 
- Include quote from Dogbe (Requested)
- Include quote from PyLadies
- Include quote from BPD Executor (to be provided)

### PR DESCRIPTION
**Pull request by anoexpected**
… at PyCon 2024

### CODE REVIEW

**Comment by kjaymiller** [comment]
Great work. Let me get you the quotes that you need

**Comment by anoexpected** [comment]
> Great work. Let me get you the quotes that you need

It's ok

File: _posts/2024-05-25-abigail-mesrenyame-dogbe-wins-inaugural-outstanding-pylady-award.md [hunk 1]
```diff
@@ -1,16 +1,23 @@
...
 layout: post
 lang: en
-title: "Abigail Mesrenyame Dogbe Honored with Outstanding PyLady Award at PyCon 2024"
+title: "Abigail Mesrenyame Dogbe Honored with Inaugural Outstanding PyLady Award"
 date: 2024-01-25
 ---
+<div style="display: flex; align-items: flex-start;">
 
-Every year, the PyLadies community recognizes individuals for their remarkable contributions to the open-source community with the Outstanding PyLady award. This year, we are thrilled to announce that [Abigail Mesrenyame Dogbe](https://mesrenyamedogbe.hashnode.dev/), a dedicated member of Black Python Devs, has been honored with this prestigious award at PyCon 2024 in Pittsburgh, PA.
+<img src="https://mesreandfriends.org/wp-content/uploads/2024/03/Dogbe_Abigail-DSC07499-1-1-1.jpg" style="margin-right: 20px; width: 100px; height: 100px; border-radius: 50%;" alt="Abigail Mesrenyame Dogbe">
```
Comment (kjaymiller) [inline _posts/2024-05-25-abigail-mesrenyame-dogbe-wins-inaugural-outstanding-pylady-award.md]: I'm a little worried about the size of this image being a little small. Any chance that we can use either of these images.

 - www/_assets/images/blog/opa_winners_2024.png from the Pyladies Post - https://github.com/pyladies/pyladies/pull/664/files#diff-8e7d78d3423230e223108a6ba66bc3117118e69807be64c5e69c695530eb6af3

Image (Please credit in alt) - https://x.com/takanory/status/1792299062248591644/photo/1

**Review by kjaymiller — CHANGES_REQUESTED** [review]
I left a comment in about the image. I think this is good to go after that and we'll wait until Pyladies makes their announcement and then we'll publish alongside of it.

**Comment by anoexpected** [comment]
> I left a comment in about the image. I think this is good to go after that and we'll wait until Pyladies makes their announcement and then we'll publish alongside of it.

It's ok I saw the comment so I will change the image with the one you suggested and make it big right?

**Comment by kjaymiller** [comment]
@anoexpected - we're still waiting on pyladies to make their post public

**Comment by kjaymiller** [comment]
And it's live!

And oddly dated back IN MAY. We can go ahead and merge

**Pull request merged**